donut/modules/newsgroups/routes.py

```diff
@@ -178,4 +179,16 @@ def posting_positions(group_id):
     if 'username' not in flask.session:
         return flask.abort(403)
     user_id = auth_utils.get_user_id(flask.session['username'])
-    return flask.jsonify(helpers.get_posting_positions(group_id, user_id))
+    if auth_utils.is_admin():
+        pos = [{
+            'pos_id': groups.get_position_id('Devteam', 'Devteam Member'),
+            'pos_name': 'Devteam Member'
+        }]
+    pos.extend(helpers.get_posting_positions(group_id, user_id))
+    return flask.jsonify(pos)
+
+
+def get_user_actions(user_id):
```

Comment:
I'm not a big fan of having two identically named functions. Is there a reason not to include the admin check in `helpers.get_user_actions()`? If the original behavior of `helpers.get_user_actions()` needs to be preserved, let's rename this function and move it to `helpers`.

Reply:
Because the unit tests for helper functions can't call auth_utils.is_admin()